### DISCUSSION TITLE
Can Not Disable “Hold to double speed”

### DISCUSSION THREAD

**Discussion by ahamza61** [PoomSmart/YTABConfig#95] [General]
I was  able to hold and seek video but on latest versions when I tap and hold video plays 2x for a while. How can I disable this?

**Comment by ExpectationChen** [comment]
YTColdConfig.speedMasterArm2FastForwardWithoutSeekBySliding
You can try to turn this off, then you can use "slide to seek".

**Reply by qnblackcat** [reply]
nice catch
